### PR TITLE
Fix/issue#421: Added message for case when user click on confirmation link not first time

### PR DESCRIPTION
**Pull request by alexbojko**
fix(issue#421): Added message for case when user click on confirmation link not first time
    If user click link first time (logged in/not logged in - pipeline is logging user in) - it will log him in and redirect to profile page
    If user click more than 1 time on confirmation link - if logged in - will be redirected to profile page, if not - to login page.

### CODE REVIEW

File: mysite/psa/views.py [hunk 1]
```diff
@@ -7,6 +8,11 @@
 from django.contrib.auth import logout, login, authenticate
 from social.backends.utils import load_backends
 
+from social.apps.django_app.views import complete as social_complete
+from social.exceptions import AuthMissingParameter
+from ct.forms import LogoutForm
```
Comment (maxsocl) [inline mysite/psa/views.py]: PEP8 formatting for imports:
https://www.python.org/dev/peps/pep-0008/#imports
```
Imports should be grouped in the following order:

standard library imports
related third party imports
local application/library specific imports
```

File: mysite/mysite/urls.py [hunk 2]
```diff
@@ -29,6 +36,8 @@
 
 
     url(r'^email-sent/$', 'psa.views.validation_sent'),
+    url(r'^complete/(?P<backend>[^/]+){0}$'.format(extra), complete,
```
Comment (maxsocl) [inline mysite/mysite/urls.py]: I thought we decided to handle manually only email backend.

**Comment by maxsocl** [comment]
@alexbojko Also tests for this case required.

File: mysite/psa/views.py (hunk 1 above)
Comment (alexbojko) [inline mysite/psa/views.py]: sure, it's my fault. fixing  it now

**Pull request merged**